File: Ct/void alternateSort(int arr[], int LENGTH)/solution1.java
import java.util.Scanner;

public class Hello {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int N = sc.nextInt();
    printPattern(N);
  }
  private static void alternateSort(int arr[], int LENGTH) {
    Arrays.sort(arr);
    int x[]=new int[LENGTH];
    int i=0,l=0,r=LENGTH-1;
    for(l=0,r=-1+LENGTH;l<r;r--,l++){
        x[i++]=arr[r];
        x[i++]=arr[l];
    }
    if(LENGTH%2==1)
    x[r+l]=arr[r];
    for(i=0;i<LENGTH;i++)
    arr[i]=x[i];
  }
} // end of class

File: Ct/void alternateSort(int arr[], int LENGTH)/solution1.c
#include <stdio.h>
void alternateSort(int arr[], int LENGTH)
{
    for(int i=0;i<LENGTH;i++){
        for(int j=i+1;j<LENGTH;j++){
            if(arr[i]>arr[j]){
                int t=arr[i];
                arr[i]=arr[j];
                arr[j]=t;
            }
        }
    }
    int a[LENGTH];
    for(int i=0;i<LENGTH;i++) a[i]=arr[i];
    int i=0,j=LENGTH-1,ctr=0;
    while(i<j){
        arr[ctr++]=a[j];
        arr[ctr++]=a[i];
        i++;
        j--;
    }
    if(LENGTH%2) arr[ctr]=a[i];
}
int main()
{
    int N;
    scanf("%d",&N);
    int arr[N];

    int index;
    for(index = 0; index < N; index++)
    {
        scanf("%d",&arr[index]);
    }

    alternateSort(arr,N);

    for(index = 0; index < N; index++)
    {
        printf("%d ",arr[index]);
    }
}
